Extension/src/LanguageServer/configurations.ts

```diff
@@ -425,6 +425,10 @@ export class CppProperties {
         result["workspaceFolderBasename"] = this.rootUri ? path.basename(this.rootUri.fsPath) : "";
         result["execPath"] = process.execPath;
         result["pathSeparator"] = (os.platform() === 'win32') ? "\\" : "/";
+        result["/"] = (os.platform() === 'win32') ? "\\" : "/";
+        result["userHome"] = os.homedir();
+        result["vcpkgRoot"] = util.getVcpkgRoot();
```

Comment:
In the case where `util.VcpkgRoot()` does not return anything, the variable should not be replaced with "".

Do something like this instead:
```ts
if (util.getVcpkgRoot()) {
    result["vcpkgRoot"] = util.getVcpkgRoot();
}
```